### ISSUE TITLE
julia_command

### ISSUE THREAD

**Issue by galaxywall** [JuliaInterop/JuliaCall#138]
Could you please tell me how I am supposed to correctly read in a CSV file using julia_command() or some other function in JuliaCall? I tried the command **julia_command("using CSV; CSV.read('data.csv');")** and got the error below. I suspect that the error is due to the quotation marks.

**Error: Error happens in Julia.
Base.Meta.ParseError("character literal contains multiple characters")
Stacktrace:
 [1] parse(::String, ::Int64; greedy::Bool, raise::Bool, depwarn::Bool) at ./meta.jl:184
 [2] parse(::String; raise::Bool, depwarn::Bool) at ./meta.jl:215
 [3] parse at ./meta.jl:215 [inlined]
 [4] eval_string(::String) at /home/galaxywall/R/x86_64-pc-linux-gnu-library/4.0/JuliaCall/julia/setup.jl:195
 [5] docall(::Ptr{Nothing}) at /home/galaxywall/R/x86_64-pc-linux-gnu-library/4.0/JuliaCall/julia/setup.jl:168**

**Comment by Non-Contradiction** [comment]
Yes, you are correct. The error is indeed due to the quotation marks.
In R, both `"` and `'` can be used for string. However, in Julia, `"` is for string and `'` is for char (a single character).
So we need to use `"` to wrap the filename to let Julia know this is a string.
For example,
`julia_command("using CSV; CSV.read(\"data.csv\");")`
or 
`julia_command('using CSV; CSV.read("data.csv");')` 
should both work.
